src/server.ts

```diff
@@ -1,36 +1,25 @@
-import express, {Request, Response} from 'express';
-import cookieParser from 'cookie-parser';
+import express, { Request, Response } from "express";
+import cookieParser from "cookie-parser";
 import config from "./config/config";
-import morgan from 'morgan'; // Http request logger, help debug
-import cors from 'cors';
+import morgan from "morgan"; // Http request logger, help debug
 
 // Import routes
 import authRoutes from "./routes/auth";
+import accessCodesRouter from "./routes/accessCodes";
 
 // Configuration
 const app = express();
 const HOST: string = config.HOST;
 const PORT: number = config.PORT;
 
-// Allows us to set new property `user` for Request object throughout this project
-// Ex: req.user will have info from users table, NOT from tenants table.
-declare module "express-serve-static-core" {
```

Comment:
What is the reasoning for removing the customize added `user` property to `Request`?